code/modules/organs/robolimbs.dm

```diff
@@ -42,7 +42,7 @@ var/const/standard_monitor_styles = "blank=ipc_blank;\
 	var/lifelike										 // If set, appears organic.
 	var/skin_tone										 // If set, applies skin tone rather than part color
 	var/blood_color = "#030303"
-	var/list/species_cannot_use = list(SPECIES_TESHARI, "Promethean") //VOREStation Add
+	var/list/species_cannot_use = list("Promethean")	 //VOREStation Edit
```

Comment:
n o

You need to add a reverse of this to make this acceptable (as in, a species_allowed). I won't merge anything that enables every prosthetic for Teshari, they look totally insane.

Reply:
Oh, fair enough. Didn't think of them being able to use other prosthetic limbs.

Reply:
You can just override it on at one with list("Promethean") instead if you don't want to do a whole allowed system.